protos/flow.proto

```diff
@@ -71,6 +71,8 @@ message FlowConnectionConfigs {
   string synced_at_col_name = 19;
 
   string script = 20;
+
+  bool same_same = 21;
```

Comment:
This should be changed to an enum re what type system is used (current, QValue vs PGValue)